doc-content/enterprise-only/project-data/reviewing-change-requests-proc.adoc

```diff
@@ -18,7 +18,7 @@ image::project-data/change-branch1.png[Selecting a branch menu]
 .. Review the *Overview* tab for general information about the change request.
 .. Click the *Changed Files* tab and expand a file to review the proposed changes.
 . Click a button in the top right corner.
-.. Click *Accept* to merge the changes into the target branch.
+.. Click *Squash and Merge* to merge all the changes into the target branch.
```

Comment:
The `Accept` button has been divided into two buttons:
- `Squash and Merge`: All new commits of the source branch are combined in a single one and added on top of the target branch.
- `Merge`: Merge all the changes from the source branch into the target branch.

So, in my opinion, we need to specify both operations on the doc.

**Reference**:
![squash](https://user-images.githubusercontent.com/638737/74954228-8ad23380-53e1-11ea-86a2-a4c4d0e42a81.png)

Reply:
@caponetto how about adding following bullets in the doc:

- Click **Squash and Merge** to squash all commits into a single commit and add to the target branch.
- Click **Merge** to merge all the changes into the target branch.

Reply:
@hmanwani-rh Looks good to me :)

Reply:
@caponetto Thanks for providing the inputs, have made changes as suggested.